### PR TITLE
Add serialization of dict and list to json bytes in python Kafka

### PR DESCRIPTION
**Pull request by cbornet**

### CODE REVIEW

File: langstream-runtime/langstream-runtime-impl/src/main/python/langstream_runtime/tests/test_kafka_connection.py [hunk 1]
```diff
@@ -322,6 +322,7 @@ def test_serializers():
             ("ByteArraySerializer", True, b"\x01"),
             ("ByteArraySerializer", 42, b"\x00\x00\x00\x00\x00\x00\x00\x2A"),
             ("ByteArraySerializer", 42.0, b"\x40\x45\x00\x00\x00\x00\x00\x00"),
+            ("ByteArraySerializer", {"a": "b", "c": 42.0}, b'{"a": "b", "c": 42.0}'),
```
Comment (eolivelli) [inline langstream-runtime/langstream-runtime-impl/src/main/python/langstream_runtime/tests/test_kafka_connection.py:325]: is this covering only "dict" ?

**Review by eolivelli — APPROVED** [review]
+1

great idea

File: langstream-runtime/langstream-runtime-impl/src/main/python/langstream_runtime/tests/test_kafka_connection.py (hunk 1 above)
Comment (cbornet) [reply langstream-runtime/langstream-runtime-impl/src/main/python/langstream_runtime/tests/test_kafka_connection.py:325]: Good catch! I should also cover list!

Comment (cbornet) [reply langstream-runtime/langstream-runtime-impl/src/main/python/langstream_runtime/tests/test_kafka_connection.py:325]: done

**Pull request merged**